### ISSUE TITLE
`get_json_objects` function panics during planning

### ISSUE THREAD

**Issue by mwylde** [ArroyoSystems/arroyo#340]
This query produces a panic during planning:

```sql
create table input (
    value TEXT
) WITH (
    connector = 'kafka',
    bootstrap_servers = 'localhost:9092',
    type = 'source',
    format = 'raw_string',
    topic = 'test'
);

select get_json_objects(value, '$.x')
from input;
```

```
2023-10-01T23:38:01.020469Z ERROR arroyo_server_common: panicked at 'not yet implemented', arroyo-sql/src/types.rs:1049:38 panic.file="arroyo-sql/src/types.rs" panic.line=1049 panic.column=38
```

This is because the `get_json_objects` function is returning an array, which is not supported by the parquet builder code added for the filesystem sink.

**Comment by mwylde** [comment]
This is partially fixed by #347 which enables array-creation functions that don't go directly into a sink

**Comment by edmondop** [comment]
> This is partially fixed by #347 which enables array-creation functions that don't go directly into a sink

Is the filesystem sink still to be fixed? I can look into it

**Comment by mwylde** [comment]
Fixed in 0.10
